frontend/src/components/Card/Card.tsx

```diff
@@ -1,41 +1,20 @@
-import { RUICard, Button } from "@/components/retroui";
-import { toast } from "sonner";
+import { RUICard } from "@/components/retroui";
+import { CardDetails } from "../CardDetails";
 
-interface ProductProps {
+export interface ProductProps {
```

Comment:
```suggestion
export type ProductProps {
```

We'll use `type` as the default unless we need to use the features `interface` gives us. 
An [explanation](https://www.totaltypescript.com/type-vs-interface-which-should-you-use)